### ISSUE TITLE
📄 Add Detail Pages for Definitions

### ISSUE THREAD

**Issue by ovlb** [tatianamac/selfdefined#53]
As the site grows, the homepage gets more and more crowded and impossible to scan. I think we should link to detail pages for single definitions (e.g. `/definitions/crazy` instead of `#crazy`).

Edit: I keep track of this issue and update the to-do list in #75. So have a look there for the current status

**Comment by tatianamac** [comment]
Agree with this change for sure! It was always the hope that the site would grow. 🤗 _(tries not to get weird and emo)_

We should be mindful though of future linking strategies when coming up with the canonical link. For example, one hope is to be able to concatenate terms so someone can theoretically link to serve up a page that shows all the terms they identify as:

`?queer+disabled+wheelchair-user`

So, tl;dr, let's do it, as long as we feel confident the above scenario can be addressed as we start to look at broader database needs

**Comment by ovlb** [comment]
Yip. The combined pages are definitely a part of it. I think they might even become simpler once we have these detail pages, as we can already start defining a layout :) 

My reasoning for `/definitions/word` is that the homepage might change in the future and become more of an overview introducing the whole project with its bots, API and so forth. By having an URL scope for the definitions we avoid dealing with redirects in the future when we might want to have the API info page at `/api`. 

Is it okay if I take this issue?

**Comment by tatianamac** [comment]
Yes, awesome, I'm so glad you're such a holistic thinker! 🤗

Also I should add that we need to create redirects as there are some links out in the world already for `/#word`. Should I open up a new issue for this?

**Comment by ovlb** [comment]
I think that’s an acceptance criteria for closing this issue. 

I’ll add a to-do list to the OP asap to make the scope of this issue clearer. :)

**Comment by ovlb** [comment]
@tatianamac I’ve added a to-do list. Anything else that comes to mind?

**Comment by tatianamac** [comment]
Looks good!

What is the intention of next/last links? Do they link to the next and previous words alphabetically? I worry a bit about what we mean by this when we consider sub-terms would theoretically appear twice in the full list, if that makes sense.

(For example, `cis` appears both under C (cis) and G (cisgender).)

**Comment by ovlb** [comment]
I thought it is a nice touch for the page, to encourage browsing around. Probably have thought about it because I like the game «open a dictionary and read whatever words show up so much». The Duden (german dictionary) has a similar feature at the bottom of their definition pages: https://www.duden.de/rechtschreibung/Geviert


Random feature idea: A «Show me a random definition»-button on the homepage.

**Comment by ovlb** [comment]
One solution for the sub-terms might be to include only the main term «gender» while somewhere around «ge» and cisgender while browsing around «ci». In any way, _cisgender_ will be shown in the sub-terms of gender on the detail page of _gender._

**Comment by tatianamac** [comment]
I think this a good solution for right now. I have a feeling I'm missing some edge cases right now

> One solution for the sub-terms might be to include only the main term «gender» while somewhere around «ge» and cisgender while browsing around «ci». In any way, _cisgender_ will be shown in the sub-terms of gender on the detail page of _gender._

**Comment by ovlb** [comment]
> I think this a good solution for right now. I have a feeling I'm missing some edge cases right now

There will be edge cases :D We will fix them once they poke us too hard.
